### ISSUE TITLE
[BUG] After consenting the original form is shown

### ISSUE THREAD

**Issue by matcolton** [eliashaeussler/typo3-form-consent#138]
### TYPO3 version

12.4.8

### PHP version

8.2.12

### Extension version

1.0

### Composer mode

- [X] I'm running TYPO3 in composer mode.

### Operating system

Fedora Workstation

### Current behavior

After clicking on the link in the consent mail the form is shown again. But the assigned images are missing.
If the page is reloaded the "alreadyApproved" notice is shown, yet the additional finishers of the "isConsentApproved()" variant are **not** fired.
The images are stored in the desired place, but this extension doesn't seem to be able to find them and/or reassign them.
Is this extension designed work with uploads/images?

### Expected behavior

Success notice, additional finishers of condition "isConsentApproved()" are initiated after clicking on link in consent email. 

### Steps to reproduce

- Fill out and send the form.
- Recieve email with consent link.
- Klick on link.
- Page shows form instead of resending the form automatically. Text entries exist (input[type=text], textarea, but lacking the assigned images.

### Additional context

```yaml
renderingOptions:
  submitButtonLabel: Absenden
type: Form
identifier: meineProjektXYZ
label: 'Meine ProjektXYZ'
prototypeName: standard
variants:
  -
    identifier: consent-approved
    condition: 'isConsentApproved()'
    finishers:
      -
        options:
          subject: 'Neue ProjektXYZ-Bilder'
          recipients:
            myemail@foobar.com: 'My Name'
          senderAddress: website@barfoo.de
          senderName: 'Acme XY'
          addHtmlPart: true
          attachUploads: true
          translation:
            language: Standard
          title: ''
        identifier: EmailToReceiver
finishers:
  -
    options:
      message: ''
      contentElementUid: '1014'
    identifier: Confirmation
  -
    options:      
      subject: 'Zustimmung zur Verwendung meiner Bilder in der ProjektXYZ Bildergalerie'
      recipientAddress: '{email}'
      recipientName: ''
      senderAddress: website@barfoo.de
      senderName: 'Acme XY'
      approvalPeriod: '86400'
      showDismissLink: false
      confirmationPid: '152'
      storagePid: '151'
    identifier: Consent
renderables:
  -
    renderingOptions:
      previousButtonLabel: 'Vorherige Seite'
      nextButtonLabel: 'Nächster Schritt'
    type: Page
    identifier: page-1
    label: ''
    renderables:
      -
        defaultValue: ''
        type: Email
        identifier: email
        label: E-Mail
        properties:
          fluidAdditionalAttributes:
            required: required
        validators:
          -
            identifier: EmailAddress
          -
            identifier: NotEmpty
      -
        properties:
          saveToFileMount: '1:/user_upload/Meine_ProjektXYZ/'
          allowedMimeTypes:
            - image/jpeg
            - image/jpg
          fluidAdditionalAttributes:
            required: required
          validationErrorMessages:
            -
              code: 1221560910
              message: 'Bitte gib ein Bild im JPG-Format mit maximal 4MB Größe an.'
            -
              code: 1221560718
              message: 'Bitte gib ein Bild im JPG-Format mit maximal 4MB Größe an.'
            -
              code: 1347992400
              message: 'Bitte gib ein Bild im JPG-Format mit maximal 4MB Größe an.'
            -
              code: 1347992453
              message: 'Bitte gib ein Bild im JPG-Format mit maximal 4MB Größe an.'
        type: ImageUpload
        identifier: image-1
        label: 'Bild 1'
        validators:
          -
            identifier: NotEmpty
          -
            options:
              minimum: 0B
              maximum: 4M
            identifier: FileSize
      -
        properties:
          saveToFileMount: '1:/user_upload/Meine_ProjektXYZ/'
          allowedMimeTypes:
            - image/jpeg
            - image/jpg
        type: ImageUpload
        identifier: image-2
        label: 'Bild 2'
        validators:
          -
            options:
              minimum: 0B
              maximum: 4M
            identifier: FileSize
      -
        defaultValue: ''
        type: Textarea
        identifier: additional-notes
        label: 'Das möchte ich zu meinem Bild/Lieblingsort noch sagen'
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

**Comment by eliashaeussler** [comment]
Hi @matcolton, thanks for your detailed bug report. This is now fixed and released as `1.1.0`.

**Comment by matcolton** [comment]
Fantastic, works like a charm now! Thanks!
